src/cephadm/cephadm

```diff
@@ -5866,7 +5920,9 @@ def list_daemons(ctx, detail=True, legacy_dir=None):
                                         seen_versions[image_id] = version
                                 elif daemon_type in ['prometheus',
                                                      'alertmanager',
-                                                     'node-exporter']:
+                                                     'node-exporter',
+                                                     'loki',
+                                                     'promtail']:
```

Comment:
does the same method for grabbing the version from alertmanager, node-exporter and prometheus work for loki and promtail out of the box? It will be something like "loki --version" run in the container based on this.